swagger-spec/registrations/comments_list.yaml

```diff
@@ -0,0 +1,174 @@
+# /registrations/{node_id}/comments/
+get:
+  summary: List all comments
+  description: >-
+
+    A paginated list of comments of a node's registration.
+
+
+    The returned comments are sorted by their creation date, with the most recent
+    comments appearing first.
+
+    ####Permissions
+
+    Comments of public registrations are given read-only access to everyone.
+
+
+    If the comment-level is `private`, only registration contributors have permission to comment.
+
+
+    If the comment-level is `public`, any logged-in OSF user can comment.
+
+
+    Comments of private registrations are only visible to contributors and administrators on the parent node.
+
+    ####Returns
+
+    Returns a JSON object containing `data` and `links` keys.
+
+
+    The `data` key contains an array of comment objects.
+    Each resource in the array is a separate comment object.
+
+
+    The `links` key contains a dictionary of links that can be used for [pagination](#Introduction_pagination).
+
+    #### Filtering
+
+    You can optionally request that the response only include comments that match your filters by utilizing the `filter` query parameter, e.g.
+    https://api.osf.io/v2/registrations/ezcuj/comments/?filter[target_id]=jg7sezmdnt93
```

Comment:
I think this link will break when clicked because `ezcuj` is a node, not a registration.